tests/DummyData.spec.js

```diff
@@ -8,21 +8,29 @@ beforeAll(async () => {
     await loadNimiq();
 });
 
+/** @type {{[key: string]: any}} */
 const Dummy = {};
 
+/* To generate HEX arrays, you can use [...].map(n => '0x' + n.toString(16)) from a decimal array, and then remove the quotes. */
+
 Dummy.keys = [
     // eslint-disable-next-line max-len
-    Uint8Array.from([0x68, 0x2d, 0xf0, 0x35, 0xc3, 0x0e, 0x8c, 0x55, 0xbd, 0xa4, 0xab, 0x51, 0x72, 0x13, 0x79, 0x2c, 0x2a, 0xf4, 0xb4, 0xe9, 0xc2, 0x3a, 0xc2, 0xcf, 0x11, 0xbd, 0x28, 0xb2, 0x37, 0xa6, 0x54, 0x4e]),
+    Uint8Array.from([ 0x68, 0x2d, 0xf0, 0x35, 0xc3, 0x0e, 0x8c, 0x55, 0xbd, 0xa4, 0xab, 0x51, 0x72, 0x13, 0x79, 0x2c, 0x2a, 0xf4, 0xb4, 0xe9, 0xc2, 0x3a, 0xc2, 0xcf, 0x11, 0xbd, 0x28, 0xb2, 0x37, 0xa6, 0x54, 0x4e ]),
```

Comment:
Why are you adding the extra spaces? Seems like an unnecessary change to me and I would actually prefer it the way it was before.

Reply:
Haha, that's actually for readability in my editor, where I have automatic line-breaking and with the extra space, the values align better:
![image](https://user-images.githubusercontent.com/1828163/52434399-d5ce2100-2ad4-11e9-9afd-615781b6e430.png)